tests/BenchmarkDotNet.IntegrationTests/BenchmarkDotNet.IntegrationTests.csproj

```diff
@@ -31,6 +31,12 @@
     <ProjectReference Include="..\..\src\BenchmarkDotNet.Diagnostics.dotTrace\BenchmarkDotNet.Diagnostics.dotTrace.csproj" />
     <ProjectReference Include="..\..\src\BenchmarkDotNet.Diagnostics.dotMemory\BenchmarkDotNet.Diagnostics.dotMemory.csproj" />
   </ItemGroup>
+  
+  <ItemGroup Condition="'$(TargetFramework)' == 'net8.0'">
+    <!-- This test project is only relevant for .NET 5+ projects -->
```

Comment:
This is also relevant for .Net Framework projects using sdk-style projects using `CsProjClassicNetToolchain`. You can ensure that toolchain is used in the test instead of `RoslynToolchain` by passing the toolchain in the config. You can test both `CsProjClassicNetToolchain.Net462` and `CsProjCoreToolchain.NetCoreApp80`.

Reply:
I've used .NET Core for so long that I completely forgot csproj-style projects were also made available for the .NET Framework. 😆